scripts/resultsSum.pl

```diff
@@ -244,11 +246,31 @@ sub resultReporter {
 		if ($jdkVendor ne '') {
 			$vendorParam = "&JDK_VENDOR=" . $jdkVendor;
 		}
-
-		my $rebuildLinkBase = "parambuild/?JDK_VERSION=$jdkVersion&JDK_IMPL=$jdkImpl$vendorParam$buildParam$platformParam$customTargetParam";
+		my $jenkinsParam = "";
+		if ( defined $ENV{'JCK_GIT_REPO'} ) {
+			$jenkinsParam .= "&JCK_GIT_REPO="        . $ENV{'JCK_GIT_REPO'};
+		}
+		if ( defined $ENV{'SDK_RESOURCE'} ) {
+			$jenkinsParam .= "&SDK_RESOURCE="        . $ENV{'SDK_RESOURCE'};
+		}
+		if ( defined $ENV{'CUSTOMIZED_SDK_URL'} ) {
+			$jenkinsParam .= "&CUSTOMIZED_SDK_URL="  . $ENV{'CUSTOMIZED_SDK_URL'};
+		}
+		if ( defined $ENV{'CUSTOMIZED_SDK_URL_CREDENTIAL_ID'} ) {
+			$jenkinsParam .= "&CUSTOMIZED_SDK_URL_CREDENTIAL_ID=" . $ENV{'CUSTOMIZED_SDK_URL_CREDENTIAL_ID'};
+		}
+		if ( defined $ENV{'UPSTREAM_JOB_NAME'} ) {
+			$jenkinsParam .= "&UPSTREAM_JOB_NAME="   . $ENV{'UPSTREAM_JOB_NAME'};
+		}
+		if ( defined $ENV{'UPSTREAM_JOB_NUMBER'} ) {
+			$jenkinsParam .= "&UPSTREAM_JOB_NUMBER=" . $ENV{'UPSTREAM_JOB_NUMBER'};
+		}
+		
+		my $rebuildLinkBase = "parambuild/?JDK_VERSION=$jdkVersion&JDK_IMPL=$jdkImpl$vendorParam$buildParam$platformParam$customTargetParam$jenkinsParam";
```

Comment:
The above Jenkins parameters changes are not necessary.

For rerun in Jenkins, please use the `Rerun in Grinder` link on the Jenkins job. It captures all Jenkins parameters. For example, in the Grinder https://ci.adoptopenjdk.net/job/Grinder/2701/

![image](https://user-images.githubusercontent.com/19273206/144456076-a6a9a949-49f3-4dc8-849b-83b5f5b3cf85.png)

The rerun link in TKG is added for local runs. It is an easy way for the local run users to validate/compare their change in the farm. TKG is designed to be CI agnostic and supports the minimal Jenkins parameters in the TKG layer.

Reply:
Stewart, I've checked and most of these will not work as you expect as they are not set as environment variables.  The only one that is set is JCK_GIT_REPO and even if you do put it in the list, you will still need to add the credential to access it (which is not set as an environment variable).  

You really won't gain much by this section, so it's mainly adding code that will never be invoked.  

Best to use the link available from the Jenkins page and not this link that is intended to be used only for local runs (and subsequently needs many more additional parameters to be filled in by the user).

Reply:
Based on the tests in https://github.com/adoptium/TKG/pull/253#issuecomment-1010419210 it seems to be working as I wanted it to when run through jenkins :-)